Neos.Flow/Classes/Cache/CacheFactory.php

```diff
@@ -101,8 +102,8 @@ public function __construct(ApplicationContext $context, Environment $environmen
      * @param string $cacheObjectName
      * @param string $backendObjectName
      * @param array $backendOptions
-     * @param boolean $persistent
-     * @return Frontend\FrontendInterface
```

Comment:
That isn't invalid, it just relative to the current namespace. Just for the record. :)